{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/config.py

```diff
@@ -53,20 +53,25 @@ def _(x):
 }
 """REST API for {{cookiecutter.project_shortname}}."""
 
-RECORDS_UI_ENDPOINTS = {
-    'recid': {
-        'pid_type': 'recid',
-        'route': '/records/<pid_value>',
-        'template': 'records/record.html',
-        'record_class': 'invenio_records_files.api:Record',
-    },
-    'recid_previewer': {
-        'pid_type': 'recid',
-        'route': '/records/<pid_value>/preview/<path:filename>',
-        'view_imp': 'invenio_previewer.views:preview',
-        'record_class': 'invenio_records_files.api:Record',
-    },
-}
+RECORDS_UI_ENDPOINTS = dict(
+    recid=dict(
+        pid_type='recid',
+        route='/records/<pid_value>',
+        template='records/record.html',
```

Comment:
Don't we need the `record_class` here?

Reply:
it will get a default config for the record_class if you don't specify
https://github.com/inveniosoftware/invenio-records-ui/blob/ae92367978f2e1e96634685bd296f0fd92b4da54/invenio_records_ui/config.py#L83

Reply:
And don't we need the one from [`invenio-records-files`](https://github.com/inveniosoftware/invenio-records-files/blob/a0ae9946b06fa1cdf658f316d491c2996b4787bc/invenio_records_files/api.py#L290)? At least this is what @ntarocco told me.

Reply:
`record_class='invenio_records_files.api:Record',`

Reply:
I see. Thanks!